sycl/doc/design/CommandGraph.md

```diff
@@ -215,3 +215,19 @@ Level Zero:
 
 Future work will include exploring L0 API extensions to improve the mapping of
 UR command-buffer to L0 command-list.
+
+### CUDA
+
+The SYCL Graph CUDA backend relies on the CUDA graph feature, which is the CUDA public API for batching command-groups with dependencies.
+
+UR sync-points are implemented as CUDA nodes using the CUDA Driver API.
+Dependencies between command-groups are implemented using CUDA graph dependencies.
+
+An executable CUDA Graph of the command-groups sequence is saved in the CommandBuffer to allow for efficient graph resubmission.
+
```

Comment:
We don't describe how submitting a UR command-buffer for execution maps to CUDA, this was a bad mapping for Level Zero, so even if it's an easy mapping for CUDA we should say that what that was.